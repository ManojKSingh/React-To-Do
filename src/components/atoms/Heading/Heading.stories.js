import React from 'react';
import {Heading} from './Heading';

export default {
  title: 'Atom/Heading',
  component: Heading,
};

export const heading = () => <Heading text="To Do List" />;