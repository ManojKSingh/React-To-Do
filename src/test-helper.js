import React from 'react';
import { ThemeProvider } from 'styled-components';
import renderer from 'react-test-renderer';
import themes from '../src/themes';

const theme = themes['light'];

export function renderWithTheme(component) {
  return renderer.create(
    <ThemeProvider theme={theme}>{component}</ThemeProvider>
  );
}



