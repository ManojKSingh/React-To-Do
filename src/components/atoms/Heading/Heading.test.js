import React from 'react';
import { renderWithTheme } from '../../../test-helper';
import {Heading} from './Heading';

describe('<Heading/>', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<Heading text="To Do List" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});