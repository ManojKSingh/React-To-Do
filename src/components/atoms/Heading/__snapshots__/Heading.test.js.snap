// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading/> renders correctly 1`] = `
<h2>
  To Do List
</h2>
`;
